core/src/main/java/hudson/ProxyConfiguration.java

```diff
@@ -428,7 +428,7 @@ public FormValidation doValidateProxy(
                 @QueryParameter("userName") String userName, @QueryParameter("secretPassword") Secret password,
                 @QueryParameter("noProxyHost") String noProxyHost) {
 
-            Jenkins.get().checkPermission(Jenkins.ADMINISTER);
+            Jenkins.get().checkPermission(Jenkins.MANAGE);
```

Comment:
This could be safely reverted.  We have discussed the idea of moving the proxy configuration to a more sensible place (would be done in a unrelated PR) since the proxy is not strictly used  for just plugin installation.